packages/nextjs/src/server/types.ts

```diff
@@ -0,0 +1,8 @@
+import { AuthStatus, ClerkBackendAPI } from '@clerk/backend-core';
+
+export const AuthStatusMapping = {
+  [AuthStatus.SignedIn]: 'i',
```

Comment:
🙃 I know that this is irrelevant, but maybe turn this into an 1 and 0 or even an actual enum starting from 1?

Reply:
Let's add something more descriptive and less hacky.

I've suggested using the actual JWT claims as an encoded string at it would help us and customers to debug easily.

We will also avoid doing [this](https://github.com/clerkinc/javascript/pull/357/files#diff-e0202f43b73d4d25fa9b0e4e88d07dbbc974c8c4381f1c8a2663bc3288679addR35) twice.

Reply:
This no longer applies, so resolving.